library/src/permutation/permutation_cpu_reference_impl.hpp

```diff
@@ -117,32 +112,35 @@ namespace hiptensor
             float Run(const Argument& arg)
             {
                 int  modeSize     = arg.mLengths.size();
-                auto elementCount = hiptensor::elementsFromLengths(std::vector(std::begin(arg.mLengths), std::end(arg.mLengths)));
+                auto elementCount = hiptensor::elementsFromLengths(
+                    std::vector(std::begin(arg.mLengths), std::end(arg.mLengths)));
```

Comment:
There is a build issue on SLES regarding the use of std::vector. 
Please be aware of changes in #225